src/main/java/picard/util/IntervalListTools.java

```diff
@@ -455,7 +455,7 @@ protected int doWork() {
 
         if (SCATTER_CONTENT != null) {
             final long listSize = SUBDIVISION_MODE.make().listWeight(output);
-            SCATTER_COUNT = (int) listSize / SCATTER_CONTENT;
+            SCATTER_COUNT = (int)Math.round((double) listSize / SCATTER_CONTENT);
```

Comment:
This will change behavior.  I agree that it is more correct this way, but just want to flag it.